src/Idris/AbsSyntax.hs

```diff
@@ -597,7 +598,7 @@ getHdrs :: Codegen -> Idris [String]
 getHdrs tgt = do i <- getIState; return (forCodegen tgt $ idris_hdrs i)
 
 getImported ::  Idris [(FilePath, Bool)]
-getImported = do i <- getIState; return (idris_imported i)
+getImported = idris_imported `fmap` getIState
```

Comment:
Would it not be preferable to use `<$>` instead?

Reply:
Do you know if `Util.Pretty` is used for anything?

Reply:
I can't find its use.

Reply:
Try removing the import, that's what I did IIRC. Later I added `-fwarn-unused-imports` https://github.com/idris-lang/Idris-dev/pull/4062/files#diff-5348f61755edf5c96c5e2818d0ff2ba6R10. So, it's used somewhere.